resources/css/custom.data-table.css

```diff
@@ -1,58 +1,28 @@
-#btn-add-record{
```

Comment:
remove `resources/css/custom.data-table.css` file and use inbuild styling for datatable